geometries/GeoDriverCSV.py

```diff
@@ -109,24 +109,24 @@ def GeoLoop(t_initial, t_final, step,file_name):
         limb_angle2 = limbangle_instr(UTC,Target, instr2)
         moon_boresight1 = moonboresight_instr(UTC, Target, instr1)
         moon_boresight2 = moonboresight_instr(UTC, Target, instr2)
-        sun_boresight1 = sunboresight_instr(UTC, Target, instr1)
-        sun_boresight2 = sunboresight_instr(UTC, Target, instr2)
-        ram_angle1 = ramangle_instr(UTC, Target, instr1)
-        ram_angle2 = ramangle_instr(UTC, Target, instr2)
-        asp_slit1 = boresight_for_difference(UTC, Target, galaxy_targ, instr1)
-        asp_slit2 = boresight_for_difference(UTC, Target, galaxy_targ, instr2)
+        sun_boresight1 = sunboresight_instr(UTC, instr1)
+        sun_boresight2 = sunboresight_instr(UTC, instr2)
+        ram_angle1 = ramangle(UTC, instr1)
+        ram_angle2 = ramangle(UTC, instr2)
```

Comment:
target (actually observer) can be derived programmatically from the instrument name and a few steps:
* instrument name
  * → frame name via sp.getfov(...)
    * → integer frame ID via sp.namfrm(...)
      * → observer name via sp.frinfo(...) (via FRAME_<frameid>_NAME Kernel Pool Variable)